### ISSUE TITLE
Resolve canonicals

### ISSUE THREAD

**Issue by developerdu** [aehrc/pathling#529]
I can group by a canonical (e.g. QuestionnaireResponse.questionnaire), but I need to be able to display the name or title of the questionnaire to make it understandable to the end user.

**Comment by johngrimes** [comment]
I agree that this would be a great feature - I think it is a duplicate of #338.
